cmd/trust-manager/app/options/options.go

```diff
@@ -162,6 +209,16 @@ func (o *Options) addBundleFlags(fs *pflag.FlagSet) {
 		"Filter expired certificates from the bundle.")
 }
 
+func (o *Options) addLoggingFlags(fs *pflag.FlagSet) {
+	fs.Var(&o.log.format,
+		"log-format",
+		"Log format (text or json)")
+
+	fs.IntVarP(&o.log.level,
+		"log-level", "v", 1,
+		"Log level (1-5).")
+}
```

Comment:
suggestion (non-blocking): I think changing the type of this arg is a breaking change.

Currently with the latest release of trust-manager I can pass `--log-level=foo5` and it'll silently ignore the invalid number and default to log level 1.

On this branch, if I pass the same arg it fails:

```text
error: invalid argument "boo1" for "-v, --log-level" flag: strconv.ParseInt: parsing "foo1": invalid syntax
```

I think one option to avoid the break would be to keep this as a string and add `func (lo *logOptions) LogLevel() (int, error)` for getting the log level, so we could handle the error ourselves (and probably just default to log level 1 and log a non-fatal error). I don't think that's required but I think it'd be nice. Your call!

Reply:
Thanks! Good catch! I will revert the change of type.

Reply:
Fixed now, but I think we should accept this breaking change. Who would ever set `--log-level=foo5`? 😉 I consider changing the type as a bugfix. Was the existing behavior (accepting any string) intentional?

Reply:
This was discussed in todays stand-up, and we reached a consensus of considering the switch of `log-level` CLI arg type from string to int a bugfix.